### ISSUE TITLE
Regression in Metabase

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#49208]
**Is your feature request related to a problem? Please describe.**
Now that we can write back into some databases we could do some regression analysis with some ML libraries out there.

**Describe the solution you'd like**
Let's say that I have a time series data set, e.g. how much I've been selling per day: I would like Metabase to tell me how much I should be selling in the next X days. I will define a training set (e.g. total of sales from the last few years) and test set (how much I've been selling this year) and a field or a set of fields that will act as features

**Describe alternatives you've considered**
Doing this outside of Metabase and then writing back

**How important is this feature to you?**
Could be cool

**Additional context**
NA
